File: src/test/java/com/larryhsiao/clotho/color/RedIntTest.java
package com.larryhsiao.clotho.color;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

/**
 * Test for {@link RedInt}
 */
public class RedIntTest {
    /**
     * Normal case
     */
    @Test
    void normalCase() {
        assertEquals(
            160,
            new RedInt(0xA0ffff).value()
        );
    }
}

File: src/test/java/com/larryhsiao/clotho/color/RedIntTest.kt
package com.larryhsiao.clotho.color

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class RedIntTest {
    @Test
    fun simple() {
        assertEquals(
            160,
            RedInt(0xA0FFFF).value()
        )
    }
}